tests/foreman/ui/test_operatingsystem.py

```diff
@@ -72,13 +72,13 @@ def test_create_os_with_different_names(self):
                     self.assertIsNotNone(self.operatingsys.search(name))
 
     @run_only_on('sat')
-    def test_create_os_with_random_params(self):
+    @tier1
+    def test_create_with_random_param(self):
```

Comment:
Missing `positive` part. Also, all the parameters are tested here, so i'd better named it `test_positive_create` and that's it.